src/common/PatchDB.cpp

```diff
@@ -1484,7 +1484,8 @@ std::string PatchDB::sqlWhereClauseFor(const std::unique_ptr<PatchDBQueryParser:
         }
         break;
     case PatchDBQueryParser::LITERAL:
-        oss << "( p.search_over LIKE '%" << protect(t->content) << "%' )";
+        oss << "( p.path LIKE '%" << protect(t->content) << "%' or";
```

Comment:
Does this search the WHOLE path to the patch? That is probably overkill.

Reply:
Because you know you can search categories by using the CAT= or CATEGORY= entry? So it's pretty easy to just do CAT=Pad and it will list all pads etc.

Reply:
Ahh excellent point yes this will make every user patch match “Documents”

Reply:
i think the approach proposed here is a bit  more broaden than CAT=. I know it may be rare, but we may have something named XXX Key, which is not under Keys cat.  By attacking this via search_for, we get this guy. This wont eliminate, CAT=. Furthermore, having the user to type cat= (or SQL like statements) is not so friendly and straightforward, as just only type the wildcard match, as here proposed. Both approaches can coexist.

Reply:
Right I think the trick is the full path in the default search. Perhaps what we should do is store the path *from the appropriate root* in the sqldb and search in that 

so like strip off the user path and search path

Reply:
yes, something on this direction. Then an update from script is necessary, right?

Reply:
Yeah lemme jot down some comments. Reading code now.